app/models/UpdateGoodsRecord.scala

```diff
@@ -56,55 +56,30 @@ object UpdateGoodsRecord {
 
   def buildGoodsDescription(
```

Comment:
we should change these function names to validateX and not buildX as we are not actually building anything

Reply:
Updated